.storybook/main.js

```diff
@@ -141,7 +142,7 @@ module.exports = {
 								options: {
 									implementation: require("postcss"),
 									postcssOptions: {
-										config: resolve(__dirname, "postcss.config.js"),
+										config: resolve(__dirname, "../postcss.config.js"),
```

Comment:
Importing the root-level postcss.  Eventually want to change this to load whatever postcss config is closest to the component.